src/main/resources/assets/js/graphql/principal/ListTypesRequest.ts

```diff
@@ -13,6 +13,12 @@ type ListTypeData = {
 export class ListTypesRequest
     extends ListItemsRequest<BucketAggregation> {
 
+    constructor(){
+        super();
+        // No need to go through everything since the only data used in the response is from aggregations
+        this.setCount(1);
```

Comment:
why now 0 ?

Reply:
@rymsha 

I asked the same thing to myself, but couldn't get the request to work with `count = 0` and 100k users locally.

Here's what happened:

I've created a script to register 100k users locally. After that the app users wasn't behaving as expected and ListTypesRequest was one of the problems. 

This request is executed when the principal browse filter panel is expanded. With 100k users and without any changes in code, we'll get this in chrome console when expanding the panel:

![image](https://user-images.githubusercontent.com/67838246/167439012-50bdaa81-95f3-4b8f-9d41-2e7adacdd060.png)

Since the only data used in the response of this request is from aggregations, no need to go through everything, so I set count to zero, and also changed [here](https://github.com/enonic/app-users/blob/master/src/main/resources/assets/js/graphql/ListGraphQlRequest.ts#L36) to be `this.count >= 0`, since that would ensure that the request was going to be executed with `count = 0`. Even with that, I was getting the same log errors in the image above.

By setting `count = 1` the request was properly executed and the aggregations returned the expected data.

Reply:
could it be some code later on converting 0 into -1 ?

Reply:
I'll investigate it further.

Reply:
@rymsha  couldn't find anywhere else than [here](https://github.com/enonic/app-users/blob/master/src/main/resources/assets/js/graphql/ListGraphQlRequest.ts#L36) that is avoiding to set the `count` or converting it from `0` into `-1`.

Reply:
That maybe it. 0 or less means "use default". And perhaps -1 is default. -1 means fetch everything

Reply:
Sadly, that's not it... as I mentioned before:

> ...so I set count to zero, and also changed [here](https://github.com/enonic/app-users/blob/master/src/main/resources/assets/js/graphql/ListGraphQlRequest.ts#L36) to be this.count >= 0, since that would ensure that the request was going to be executed with count = 0. Even with that, I was getting the same log errors in the image above.

Reply:
What @rymsha means is that even when you explicitly send `count=0` to XP, it would treat it as -1. We are investigating.

Reply:
Looked at the code and have found there is not only `this.count >= 0` check which prevents sending count eq 0 to the backend.  

`var count = params.count == null ? MAX_COUNT : params.count;`

`var count = env.args.count || Number.MAX_SAFE_INTEGER;`

Reply:
@rymsha 

Thank you @vbradnitski, couldn't find  those cases before, my bad!

`var count = env.args.count || Number.MAX_SAFE_INTEGER;` was the reason for not being able to set count to zero in some cases. It is now fixed in [this PR](https://github.com/enonic/app-users/pull/1051).